### ISSUE TITLE
decrypt works on some data files but not others

### ISSUE THREAD

**Issue by fuzzydowner** [Lyle-Tafoya/Omega#37]
I've compiled the encryption and decryption codes, but I've used only the decryption program so far.  For me, it works on some of the data files but not others.  Villages, court, abyss work fine.  The mazes don't.  The city and country dats begin to format properly but seem to time out before finishing.  I've attached one of each.  
[maze1_dec.txt](https://github.com/Lyle-Tafoya/Omega/files/14486846/maze1_dec.txt)
[country_dec.txt](https://github.com/Lyle-Tafoya/Omega/files/14486851/country_dec.txt)
[village6_dec.txt](https://github.com/Lyle-Tafoya/Omega/files/14486878/village6_dec.txt)

**Comment by Lyle-Tafoya** [comment]
Yes, I'm seeing the same issue. Not sure what could be causing this, since I haven't touched the maze files or the decryption code. I'll see if I can track down the issue, but it's not a high priority for me right now. I'll probably start by looking at older versions of Omega and seeing if the issue can be reproduced there.

**Comment by Lyle-Tafoya** [comment]
I looked at this more in depth today, and what I found is that the formatting of the maze files was intentional. They lacked newlines to make it easier to parse when loading them. I suppose anyone editing a maze file was expected to resize their text editor to be only 22 columns wide. It's only a minor annoyance, but I went ahead and changed them to contain newlines so as to avoid any confusion in the future.
As for the city and country dats, I am unable to reproduce the issue you experienced. They decrypt without issue and appear properly formatted.
I'm going to close this issue, but you can feel free to reopen if you are convinced there is still something wrong here.
